spec/validations/svg_spec.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+describe ZendeskAppsSupport::Validations::Svg do
+  let(:clean_markup) { "<svg viewBox=\"0 0 26 26\" id=\"zd-svg-icon-26-app\" width=\"100%\" height=\"100%\"><path fill=\"none\" stroke=\"currentColor\" stroke-linejoin=\"round\" stroke-width=\"2\" d=\"M4 8l9-5 9 5v9.7L13 23l-9-5.2zm9 5L4 8m9 5l9-5m-9 5v10\"></path></svg>\n" }
```

Comment:
Any reason not to use single quotes here to avoid all the back-slashes? Could also use `%Q` to avoid the line break character. See https://ruby-doc.org/core-2.1.1/doc/syntax/literals_rdoc.html#label-Strings.

Reply:
Rubocop doesn't like `%q` unless you're mixing double and single-quotes, but `%` cleans up the backslashes - thanks. Can't seem to work out how to get rid of the trailing line-break char though.